client/src/modules/reports/generate/account_report/account_report.html

```diff
@@ -22,30 +22,51 @@ <h3 class="text-capitalize" translate>REPORT.REPORT_ACCOUNTS.TITLE</h3>
 
         <div class="panel-body">
           <form name="ConfigForm" bh-submit="ReportConfigCtrl.preview(ConfigForm)" novalidate>
+
+
+
             <!-- account selection  -->
-            <bh-account-select
-              id="account-id"
-              account-id="ReportConfigCtrl.account"
-              label="FORM.SELECT.ACCOUNT"
-              name="account"
-              on-select-callback="ReportConfigCtrl.selectAccount(account)">
-            </bh-account-select>
 
-            <!-- included Date interval  -->
-            <div class="checkbox">
-              <label>
-                <input type="checkbox" ng-model="ReportConfigCtrl.dateInterval" ng-true-value="1" ng-false-value="0">
-                <span translate>FORM.LABELS.INCLUDE_DATE_INTERVAL</span>
-              </label>
+            <div class="form-group"
```

Comment:
The `bh-account-select` component already has a form-group error class as well as an `ng-messages` block. We do not need a second version here. It would be good to investigate why the ng-message and error validation are not working for the component in this context.